releasenotes/notes/1.4/prepare-1.4.0-aee22d10333374c8.yaml

```diff
@@ -0,0 +1,208 @@
+    and will certainly not be stored in a circuit.  This is due to the move of the data model 
+    to the Rust space to improve performance.
+  - |
+    The ``dag`` optional parameter in the constructor of :class:`DAGNode`
+    subclasses (namely :class:`DAGOpNode`), 
+    which has been unused and ignored since Qiskit v1.2.0, is now deprecated as of 
+    Qiskit v1.4.0 and will be removed in Qiskit v2.0.0.
+  - |
+    The Multiple-Control-Multiple-Target (MCMT) class in 
+    :class:`~qiskit.circuit.library.generalized_gates.MCMT` is now deprecated
+    and replaced by :class:`.MCMTGate`, which is a proper :class:`.Gate` subclass. Using
+    a gate instead of a circuit allows the compiler to reason about the object at a higher
+    level of abstraction and unlocks the use of multiple synthesis plugins.
+  - |
+    The :mod:`qiskit.circuit.classicalfunction` module, and with it the
+    :class:`.ClassicalFunction` class and its related :func:`.classical_function`
+    and :class:`.BooleanExpression` utilities, have been deprecated as of 
+    Qiskit v1.4.0 and will be removed in Qiskit v2.0.0. 
+    
+    This change is performed to avoid a 
+    dependency on the external library ``tweedledum``, which is no longer 
+    compatible with all of Qiskit's supported platforms and Python versions.
+    For a similar functionality please use the :class:`.PhaseOracle` which
+    is going to have a ``tweedledum``-independent implementation, and
+    the ``BitFlipOracle`` which will be added in Qiskit v2.0.0.
+
+    Until ``BitFlipOracle`` is added, a phase-flip oracle can be converted
+    to a bit-flip oracle by conditioning it on the result qubit, and applying
+    Hadamard gates before and after the application of the oracle, as in the
+    following example (where the oracle is on ``qr_x`` and the result is on ``qr_y``):
+
+    .. code-block:: python
+    
+      from qiskit import QuantumRegister, QuantumCircuit
+      from qiskit.circuit.library.phase_oracle import PhaseOracle
+
+      bool_expr = "(x0 & x1 | ~x2) & x4"
+      qr_x = QuantumRegister(4, "x")
+      qr_y = QuantumRegister(1, "y")
+
+      bit_flip_oracle = QuantumCircuit(qr_x, qr_y)
+      phase_flip_oracle = PhaseOracle(bool_expr)
+      controlled_phase_flip_oracle = phase_flip_oracle.control(1)
+      bit_flip_oracle.h(qr_y)
+      bit_flip_oracle.compose(controlled_phase_flip_oracle, qubits=[*qr_y, *qr_x], inplace=True)
+      bit_flip_oracle.h(qr_y)
+
+      print(bit_flip_oracle)
+    
+    Which results in
+
+    .. code-block:: text
+    
+                ┌───────────────┐     
+      x_0: ─────┤0              ├─────
+                │               │     
+      x_1: ─────┤1              ├─────
+                │  Phase Oracle │     
+      x_2: ─────┤2              ├─────
+                │               │     
+      x_3: ─────┤3              ├─────
+          ┌───┐└───────┬───────┘┌───┐
+       y: ┤ H ├────────■────────┤ H ├
+          └───┘                 └───┘
```

Comment:
We could use the plot directive for this instead. and get a graphical visualization. But it's not critical.